databricks/koalas/series.py

```diff
@@ -4324,6 +4324,58 @@ def pct_change(self, periods=1):
 
         return self._with_new_scol((scol - prev_row) / prev_row)
 
+    def combine_first(self, other):
+        """
+        Combine Series values, choosing the calling Series's values first.
+
+        .. note:: This API internally performs a join operation which can be pretty expensive
+            in general. if you want to use though, set `compute.ops_on_diff_frames` to True.
+
+        Parameters
+        ----------
+        other : Series
+            The value(s) to be combined with the `Series`.
+
+        Returns
+        -------
+        Series
+            The result of combining the Series with the other object.
+
+        See Also
+        --------
+        Series.combine : Perform elementwise operation on two Series
+            using a given function.
+
+        Notes
+        -----
+        Result index will be the union of the two indexes.
+
+        Examples
+        --------
+        >>> from databricks.koalas.config import set_option, reset_option
+        >>> set_option("compute.ops_on_diff_frames", True)
+        >>> s1 = ks.Series([1, np.nan])
+        >>> s2 = ks.Series([3, 4])
+        >>> s1.combine_first(s2)
```

Comment:
We don't have to perform `compute.ops_on_diff_frames` when two series are from the same dataframe.

Reply:
It seems we still need this?? 😮

```python
>>> kdf
   A  B
0  1  3
1  2  4
>>> kdf.A.combine_first(kdf.B)
...
ValueError: Cannot combine the series or dataframe because it comes from a different dataframe. In order to allow this operation, enable 'compute.ops_on_diff_frames' option.
```

Reply:
It seems we still need this. maybe i missed something??

```python
>>> kdf = ks.DataFrame({'A':[1, 2], 'B': [3, 4]})
>>> combine_frames(kdf.A, kdf.B)
...
ValueError: Cannot combine the series or dataframe because it comes from a different dataframe. In order to allow this operation, enable 'compute.ops_on_diff_frames' option.
```

Reply:
ah, i got what you meant. thanks!